lib/default-theme/NavLinks.vue

```diff
@@ -20,15 +34,18 @@
 
 <script>
 import OutboundLink from './OutboundLink.vue'
+import NavLink from './NavLink.vue'
 import { isActive, ensureExt } from './util'
 
 export default {
-  components: { OutboundLink },
+  components: { OutboundLink, NavLink },
   computed: {
     userLinks () {
-      return (this.$site.themeConfig.nav || []).map(item => ({
-        text: item.text,
-        link: ensureExt(item.link)
+      return (this.$site.themeConfig.nav || []).map(({ text, link, type, items }) => ({
+        text,
+        type,
+        link: link ? ensureExt(link) : void 0,
```

Comment:
maybe something more readable like `null` instead of `void 0`

Reply:
fixed at 50689ea13da744b0fdac8368e815fd2e6bdd7a30, thanks